File: tests/fixtures/sample.java
import java.util.List;
import java.util.ArrayList;

public class Calculator {
    private double value;

    public Calculator(double value) {
        this.value = value;
    }

    public double add(double x) {
        return this.value + x;
    }

    public static void main(String[] args) {
        Calculator calc = new Calculator(10.0);
        double result = calc.add(5.0);
        List<String> list = new ArrayList<>();
        System.out.println(result);
    }
}

File: tests/fixtures/sample.scala
import scala.collection.mutable.ListBuffer
import scala.util.Try

class Calculator(val value: Double) {
  def add(x: Double): Double = value + x

  def subtract(x: Double): Double = value - x
}

object Main {
  def main(args: Array[String]): Unit = {
    val calc = new Calculator(10.0)
    val result = calc.add(5.0)
    println(result)
  }
}
